documentation/reference_doc/4_9_hvac.tex

```diff
@@ -238,15 +238,73 @@ \subsection{Air-Side Energy Recovery}
+
+\begin{figure}
+  \centering \includegraphics[width=0.7\textwidth]{figures/econ_damper_fault.png}
+  \caption{Economizer damper stuck closed fault description.}
+  \label{fig:econ_damper_fault}
+\end{figure}
+
+Figure~\ref{fig:econ_damper_fault_single_model} shows example simulation results for a building with and without the damper fully closed fault. The fault was imposed once during the entire April period resulting in 1.8\% mechanical load increase. As mentioned previously, the energy impact of the fault can either be increased or decreased energy consumption, and Figure~\ref{fig:econ_damper_fault_single_model}(c) highlights the transition from negative to positive savings when the outdoor air temperature transitions from favorable to unfavorable conditions.
+
+\begin{figure}
+  \centering \includegraphics[width=0.7\textwidth]{figures/econ_damper_fault_single_model.png}
+  \caption{Economizer damper stuck closed fault impact on simulation results.}
+  \label{fig:econ_damper_fault_single_model}
+\end{figure}
+
+Although faults (e.g., damper fully closed) are implemented with fixed prevalence (e.g., 35\%), the actual percentage of economizers being faulted (among applicable economizers) is less than the defined prevalence due to implementation limitations. For example, 35\% of randomly selected buildings that include certain HVAC system types (categorized by the air system) are assigned the damper fully closed fault. However, certain portions of these air systems do not have economizers, thus, they cannot have an economizer fault. In other words, the current limitation is that the random selection of faulted economizers is not fully aligned with buildings that actually have economizers. In these cases, we are losing the opportunity for applying faults and decreasing the representation of fault prevalence in final building stock. Additional to sampling limitations, another limitation is that the economizer faults are currently not applied to packaged single zone systems due to incorrect applicability implementation. Newer versions of California's Title 24 energy code requires fault detection and diagnostics (FDD) for economizers which should prevent and mitigate faults. However, ComStock does not reflect the impact of FDD technology, possibly overestimating the impact of faults in buildings with newer HVAC systems in California.
```

Comment:
My goal for this cycle is to resolve this below. I think we can remove this sentence assuming I'll resolve this in this cycle. But I defer that decision to you.

> Additional to sampling limitations, another limitation is that the economizer faults are currently not applied to packaged single zone systems due to incorrect applicability implementation.

Reply:
I believe in you! removed this limitation.